JavaScript-Bits/nodejs-crud-api
# add mocha, chai and endpoint test cases for /elections

#### DESCRIPTION

- Added layout for e2e test cases
- Added 2 test cases for `/elections` endpoint
- Added mocha command to run test cases via `npm test`

### STEPS TO MANUAL TEST/// TESTING INSTRUCTIONS

- Install npm packages `npm install`

- Run npm test `npm test`

  - Assert test cases run to succession

  - We should not have flaky tests
